libs/src/sdk/utils/cid.ts

```diff
@@ -0,0 +1,14 @@
+import { CID } from 'multiformats/cid'
+import { encode, code } from 'multiformats/codecs/json'
+import { sha256 } from 'multiformats/hashes/sha2'
+
+/**
+ * Generates CID V1 for a JSON metadata object (NOT the string of the metadata - must be an object).
```

Comment:
Prob would be useful to explain what the CID v1 means / what its used for

Reply:
Agreed, I can add more clarity here. Copied from https://github.com/AudiusProject/audius-protocol/blob/main/libs/src/utils/fileHasher.ts#L246